spec/rubocop/cop/rails/transaction_exit_statement_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::TransactionExitStatement, :config do
+  it 'registers an offense when exit statements are detected in transactions' do
+    expect_offense(<<~RUBY)
+      ApplicationRecord.transaction do
+        return if user.active?
+        ^^^^^^ Exit statements such as `return`, `break` and `throw` are not allowed.
```

Comment:
It would be user-friendly to show an alternative :-)

```suggestion
        ^^^^^^ Exit statement as `return` is not allowed. Use `raise` (rollback) or `next` (commit).
```